saga-demo/README.md

```diff
@@ -24,9 +24,9 @@ User ---> Saga -----> flight booking service
 ```
 
 ## Running Demo
-1. run the following command to create docker images in saga project root folder
+1. when Saga service is ready, run the following command in saga/saga-demo/ directory to create microservice docker images.
```

Comment:
perhaps it's better to ask the user to create docker images in saga root folder with command
`mvn package -DskipTests -Pdocker -Pdemo`

Reply:
modified completed